src/stores/pacbioPoolCreate.js

```diff
@@ -594,18 +600,20 @@ export const usePacbioPoolCreateStore = defineStore('pacbioPoolCreate', {
      * // Populate used_aliquots from a pool
      * const result = await populateUsedAliquotsFromPool(1);
      * console.log(result); // { success: true, errors: [] }
+     * This method actually fetches the pool so should be renamed. It was difficult to work out which method created the pool,
+     * fetchPool
      */
```

Comment:
Good point. Do you think it is good to rename as part of this PR?

Reply:
I think the best thing to do would be to refactor it and move the post fetch processing elsewhere.

Probably not something I want to do as part of this.

We can talk about it when we do some aliquot refactoring.